### PR TITLE
Print instruction after teardown

### PR DESCRIPTION
**Pull request by honnix**
# TL;DR

Print instruction after execution sandbox/demo teardown command.

This PR also includes a doc build fix similarly as https://github.com/flyteorg/flytekit/pull/1619.

## Type
- [x] Bug Fix
- [x] Feature
- [ ] Plugin

## Are all requirements met?

- [x] Code completed
- [x] Smoke tested
- [x] Unit tests added
- [ ] Code documentation added
- [ ] Any pending items have an associated Issue

## Complete description
As described in https://github.com/flyteorg/flyte/issues/3689, it is helpful to guide users cleaning up.

## Tracking Issue
Closes https://github.com/flyteorg/flyte/issues/3689

## Follow-up issue
_NA_

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -72,8 +72,14 @@ func SetupFlyteDir() error {
 	return nil
 }
 
-// PrintDemoMessage will print sandbox success message
+// PrintDemoMessage will print demo start success message
+// Deprecated: use PrintDemoStartMessage
```
Comment (honnix) [inline pkg/util/util.go:76]: If we are confident this is not used by any potential users, it is fine to delete.

File: pkg/util/util.go [hunk 2]
```diff
@@ -97,8 +102,14 @@ func PrintDemoMessage(flyteConsolePort int, kubeconfigLocation string, dryRun bo
 	fmt.Printf("%s The Minio API is hosted on localhost:30002. Use http://localhost:30080/minio/login for Minio console\n", emoji.OpenFileFolder)
 }
 
-// PrintSandboxMessage will print sandbox success message
+// PrintSandboxMessage will print sandbox start success message
+// Deprecated: use PrintSandboxStartMessage
```
Comment (honnix) [inline pkg/util/util.go:106]: Same here.

File: pkg/util/util_test.go [hunk 3]
```diff
@@ -27,9 +27,15 @@ func TestSetupFlyteDir(t *testing.T) {
...
 
-func TestPrintSandboxMessage(t *testing.T) {
+func TestPrintSandboxStartMessage(t *testing.T) {
 	t.Run("Print Sandbox Message", func(t *testing.T) {
-		PrintSandboxMessage(SandBoxConsolePort, docker.SandboxKubeconfig, false)
+		PrintSandboxStartMessage(SandBoxConsolePort, docker.SandboxKubeconfig, false)
+	})
+}
+
+func TestPrintSandboxTeardownMessage(t *testing.T) {
```
Comment (honnix) [inline pkg/util/util_test.go:36]: This test doesn't seem to be very useful, so just for the completeness.

**Comment by honnix** [comment]
```
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-prtbml3a/sphinxcontrib-yt_82d7821052bb4688b3a0669cac992342/setup.py", line 4, in <module>
          from sphinxcontrib import yt as pkg
      ImportError: cannot import name 'yt' from 'sphinxcontrib' (unknown location)
      [end of output]
```
Doesn't seem to relate to this PR.

**Comment by honnix** [comment]
> ```
>   × python setup.py egg_info did not run successfully.
>   │ exit code: 1
>   ╰─> [6 lines of output]
>       Traceback (most recent call last):
>         File "<string>", line 2, in <module>
>         File "<pip-setuptools-caller>", line 34, in <module>
>         File "/tmp/pip-install-prtbml3a/sphinxcontrib-yt_82d7821052bb4688b3a0669cac992342/setup.py", line 4, in <module>
>           from sphinxcontrib import yt as pkg
>       ImportError: cannot import name 'yt' from 'sphinxcontrib' (unknown location)
>       [end of output]
> ```
> 
> Doesn't seem to relate to this PR.

This seems to be either a pip or setuptools issue.

A successful build has `python -m pip install --upgrade --no-cache-dir pip setuptools<58.3.0`, while a failed one has `python -m pip install --upgrade --no-cache-dir pip setuptools`.

I tried pip 23.0.1 and setuptools 58.2.0 locally and that worked. https://github.com/divi255/sphinxcontrib.youtube/blob/master/setup.py#L4 has the weird usage of importing itself from setup.py and maybe pip/setuptools no longer supports that.

**Comment by eapolinario** [comment]
> > ```
> >   × python setup.py egg_info did not run successfully.
> >   │ exit code: 1
> >   ╰─> [6 lines of output]
> >       Traceback (most recent call last):
> >         File "<string>", line 2, in <module>
> >         File "<pip-setuptools-caller>", line 34, in <module>
> >         File "/tmp/pip-install-prtbml3a/sphinxcontrib-yt_82d7821052bb4688b3a0669cac992342/setup.py", line 4, in <module>
> >           from sphinxcontrib import yt as pkg
> >       ImportError: cannot import name 'yt' from 'sphinxcontrib' (unknown location)
> >       [end of output]
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Doesn't seem to relate to this PR.
> 
> This seems to be either a pip or setuptools issue.
> 
> A successful build has `python -m pip install --upgrade --no-cache-dir pip setuptools<58.3.0`, while a failed one has `python -m pip install --upgrade --no-cache-dir pip setuptools`.
> 
> I tried pip 23.0.1 and setuptools 58.2.0 locally and that worked. https://github.com/divi255/sphinxcontrib.youtube/blob/master/setup.py#L4 has the weird usage of importing itself from setup.py and maybe pip/setuptools no longer supports that.
> 
> Since this would require change to readthedocs, it is not possible to do anything in this PR.

sphinxcontrib-youtube is supposed to be a dropin replacement (we did this in other repos, e.g.: https://github.com/flyteorg/flytekit/pull/1619 )

File: pkg/util/util_test.go (hunk 3 above)
Comment (eapolinario) [reply pkg/util/util_test.go:36]: in all honesty, the previous test is not terribly useful either. It's fine to leave both there.

File: pkg/util/util.go (hunk 1 above)
Comment (eapolinario) [reply pkg/util/util.go:76]: I doubt that there's someone out there importing this. It's fine to delete.

**Comment by honnix** [comment]
> > > ```
> > >   × python setup.py egg_info did not run successfully.
> > >   │ exit code: 1
> > >   ╰─> [6 lines of output]
> > >       Traceback (most recent call last):
> > >         File "<string>", line 2, in <module>
> > >         File "<pip-setuptools-caller>", line 34, in <module>
> > >         File "/tmp/pip-install-prtbml3a/sphinxcontrib-yt_82d7821052bb4688b3a0669cac992342/setup.py", line 4, in <module>
> > >           from sphinxcontrib import yt as pkg
> > >       ImportError: cannot import name 'yt' from 'sphinxcontrib' (unknown location)
> > >       [end of output]
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Doesn't seem to relate to this PR.
> > 
> > 
> > This seems to be either a pip or setuptools issue.
> > A successful build has `python -m pip install --upgrade --no-cache-dir pip setuptools<58.3.0`, while a failed one has `python -m pip install --upgrade --no-cache-dir pip setuptools`.
> > I tried pip 23.0.1 and setuptools 58.2.0 locally and that worked. https://github.com/divi255/sphinxcontrib.youtube/blob/master/setup.py#L4 has the weird usage of importing itself from setup.py and maybe pip/setuptools no longer supports that.
> > Since this would require change to readthedocs, it is not possible to do anything in this PR.
> 
> sphinxcontrib-youtube is supposed to be a dropin replacement (we did this in other repos, e.g.: [flyteorg/flytekit#1619](https://github.com/flyteorg/flytekit/pull/1619) )

🤦 , so many different sphinxcontrib youtube libs:

* https://github.com/shomah4a/sphinxcontrib.youtube
* https://github.com/divi255/sphinxcontrib.youtube
* https://github.com/sphinx-contrib/youtube

File: doc-requirements.txt [hunk 4]
```diff
@@ -90,7 +90,7 @@ sphinxcontrib-qthelp==1.0.3
     # via sphinx
 sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
-sphinxcontrib-yt==0.2.2
+sphinxcontrib-youtube==1.2.0
```
Comment (honnix) [inline doc-requirements.txt:93]: I did this manually. I was not able to do `make doc-requirements.txt` because pip-compile complained:

```
Using legacy resolver. Consider using backtracking resolver with `--resolver=backtracking`.
Could not find a version that matches sphinx<5,<8.0.0,>=0.6,>=1.3,>=1.5.0,>=1.8,>=2,>=7.0.0,~=4.0 (from -r doc-requirements.in (line 2))
Tried: 0.1.61611, 0.1.61798, 0.1.61843, 0.1.61945, 0.1.61950, 0.2, 0.3, 0.4, 0.4.1, 0.4.2, 0.4.3, 0.5, 0.5.1, 0.5.2, 0.6, 0.6.1, 0.6.2, 0.6.3, 0.6.4, 0.6.5, 0.6.6, 0.6.7, 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 1.0.7, 1.0.8, 1.1, 1.1.1, 1.1.2, 1.1.3, 1.2, 1.2, 1.2, 1.2.1, 1.2.1, 1.2.1, 1.2.2, 1.2.2, 1.2.2, 1.2.3, 1.2.3, 1.3, 1.3, 1.3.1, 1.3.1, 1.3.2, 1.3.2, 1.3.3, 1.3.3, 1.3.4, 1.3.4, 1.3.5, 1.3.5, 1.3.6, 1.3.6, 1.4, 1.4, 1.4.1, 1.4.1, 1.4.2, 1.4.2, 1.4.3, 1.4.3, 1.4.4, 1.4.4, 1.4.5, 1.4.5, 1.4.6, 1.4.6, 1.4.7, 1.4.7, 1.4.8, 1.4.8, 1.4.9, 1.4.9, 1.5, 1.5, 1.5.1, 1.5.1, 1.5.2, 1.5.2, 1.5.3, 1.5.3, 1.5.4, 1.5.4, 1.5.5, 1.5.5, 1.5.6, 1.5.6, 1.6.1, 1.6.1, 1.6.2, 1.6.2, 1.6.3, 1.6.3, 1.6.4, 1.6.4, 1.6.5, 1.6.5, 1.6.6, 1.6.6, 1.6.7, 1.6.7, 1.7.0, 1.7.0, 1.7.1, 1.7.1, 1.7.2, 1.7.2, 1.7.3, 1.7.3, 1.7.4, 1.7.4, 1.7.5, 1.7.5, 1.7.6, 1.7.6, 1.7.7, 1.7.7, 1.7.8, 1.7.8, 1.7.9, 1.7.9, 1.8.0, 1.8.0, 1.8.1, 1.8.1, 1.8.2, 1.8.2, 1.8.3, 1.8.3, 1.8.4, 1.8.4, 1.8.5, 1.8.5, 1.8.6, 1.8.6, 2.0.0, 2.0.0, 2.0.1, 2.0.1, 2.1.0, 2.1.0, 2.1.1, 2.1.1, 2.1.2, 2.1.2, 2.2.0, 2.2.0, 2.2.1, 2.2.1, 2.2.2, 2.2.2, 2.3.0, 2.3.0, 2.3.1, 2.3.1, 2.4.0, 2.4.0, 2.4.1, 2.4.1, 2.4.2, 2.4.2, 2.4.3, 2.4.3, 2.4.4, 2.4.4, 2.4.5, 2.4.5, 3.0.0, 3.0.0, 3.0.1, 3.0.1, 3.0.2, 3.0.2, 3.0.3, 3.0.3, 3.0.4, 3.0.4, 3.1.0, 3.1.0, 3.1.1, 3.1.1, 3.1.2, 3.1.2, 3.2.0, 3.2.0, 3.2.1, 3.2.1, 3.3.0, 3.3.0, 3.3.1, 3.3.1, 3.4.0, 3.4.0, 3.4.1, 3.4.1, 3.4.2, 3.4.2, 3.4.3, 3.4.3, 3.5.0, 3.5.0, 3.5.1, 3.5.1, 3.5.2, 3.5.2, 3.5.3, 3.5.3, 3.5.4, 3.5.4, 4.0.0, 4.0.0, 4.0.1, 4.0.1, 4.0.2, 4.0.2, 4.0.3, 4.0.3, 4.1.0, 4.1.0, 4.1.1, 4.1.1, 4.1.2, 4.1.2, 4.2.0, 4.2.0, 4.3.0, 4.3.0, 4.3.1, 4.3.1, 4.3.2, 4.3.2, 4.4.0, 4.4.0, 4.5.0, 4.5.0, 5.0.0, 5.0.0, 5.0.1, 5.0.1, 5.0.2, 5.0.2, 5.1.0, 5.1.0, 5.1.1, 5.1.1, 5.2.0, 5.2.0, 5.2.0.post0, 5.2.0.post0, 5.2.1, 5.2.1, 5.2.2, 5.2.2, 5.2.3, 5.2.3, 5.3.0, 5.3.0, 6.0.0, 6.0.0, 6.0.1, 6.0.1, 6.1.0, 6.1.0, 6.1.1, 6.1.1, 6.1.2, 6.1.2, 6.1.3, 6.1.3, 6.2.0, 6.2.0, 6.2.1, 6.2.1, 7.0.0, 7.0.0, 7.0.1, 7.0.1
Skipped pre-versions: 0.5.2b1, 0.6b1, 1.0b1, 1.0b2, 1.2b1, 1.2b2, 1.2b3, 1.3b1, 1.3b1, 1.3b2, 1.3b2, 1.3b3, 1.3b3, 1.4a1, 1.4a1, 1.4b1, 1.4b1, 1.5a1, 1.5a1, 1.5a2, 1.5a2, 1.5b1, 1.5b1, 1.6b1, 1.6b1, 1.6b2, 1.6b2, 1.6b3, 1.6b3, 1.7.0b1, 1.7.0b1, 1.7.0b2, 1.7.0b2, 1.8.0b1, 1.8.0b1, 2.0.0b1, 2.0.0b1, 2.0.0b2, 2.0.0b2, 3.0.0b1, 3.0.0b1, 4.0.0b1, 4.0.0b1, 4.0.0b2, 4.0.0b2, 5.0.0b1, 5.0.0b1, 6.0.0b1, 6.0.0b1, 6.0.0b2, 6.0.0b2, 7.0.0rc1, 7.0.0rc1
There are incompatible versions in the resolved dependencies:
  sphinx (from -r doc-requirements.in (line 2))
  Sphinx<8.0.0,>=7.0.0 (from sphinx-prompt==1.7.0->-r doc-requirements.in (line 3))
  Sphinx>=0.6 (from sphinxcontrib-youtube==1.2.0->-r doc-requirements.in (line 8))
  sphinx~=4.0 (from furo==2022.4.7.dev1->-r doc-requirements.in (line 1))
  sphinx<5,>=2 (from sphinx-panels==0.6.0->-r doc-requirements.in (line 9))
  Sphinx>=1.5.0 (from sphinx_fontawesome==0.0.6->-r doc-requirements.in (line 7))
  sphinx>=1.3 (from sphinx-code-include==1.1.1->-r doc-requirements.in (line 5))
  sphinx>=2.0 (from sphinx-material==0.0.35->-r doc-requirements.in (line 4))
  sphinx>=1.8 (from sphinx-copybutton==0.5.2->-r doc-requirements.in (line 6))
make: *** [doc-requirements.txt] Error 2
```
So `sphinx-panels` and `sphinx-prompt` don't agree with each other.

Comment (honnix) [reply doc-requirements.txt:93]: `sphinx-panels` is no longer maintained https://github.com/executablebooks/sphinx-panels#sphinx-panels :(

File: docs/source/conf.py [hunk 5]
```diff
@@ -43,7 +43,7 @@
     "sphinx-prompt",
     "sphinx_copybutton",
     "sphinx_fontawesome",
-    "sphinxcontrib.yt",
+    "sphinxcontrib-youtube",
```
Comment (honnix) [inline docs/source/conf.py:46]: TIL this conf.py file.

File: docs/source/conf.py [hunk 6]
```diff
@@ -43,7 +43,7 @@
     "sphinx-prompt",
     "sphinx_copybutton",
     "sphinx_fontawesome",
-    "sphinxcontrib.yt",
+    "sphinxcontrib.youtube",
```
Comment (honnix) [inline docs/source/conf.py:46]: `sphinxcontrib-youtube` vs. `sphinxcontrib.youtube`, lol.

**Review by eapolinario — APPROVED** [review]
